holidayextras/ui-brand-heha
# HEHA-193 Update HEHA! prototype

#### What does this PR do? (please provide any background)
Adds an `example/` directory with the markup for HEHA! planner from `tripapplite` in its current state, ready for [No Divide](http://nodividestudio.com/) to pick it up and apply a redesign next week.

#### What tests does this PR have?
None

#### How can this be tested?
Please follow the new `README.md` and make sure everything makes sense. Try out the *Workflow* section locally.

#### Screenshots / Screencast
![heha prototype](https://cldup.com/k7TJNIVIMi.png)

#### What gif best describes how you feel about this work?
![](https://media0.giphy.com/media/WFEGOIIrj6SY0/200.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
